packages/babel-core/package.json

```
@@ -88,7 +88,7 @@
         "type": "module",
         "exports": {
           ".": {
-            "require": "./cjs-proxy.cjs",
+            "require": "./cjs-proxy/index.cjs",
```

The `cjs-proxy.cjs` is still out there, is it from another working feature?

---

```suggestion
            "require": "./cjs-proxy.cjs",
```

I initially moved it to a separate file and forgot to revert the change here.